arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx

```diff
@@ -249,6 +249,8 @@ contract ARBPriceConsumer is UsingTellor {
 
 You can adapt this contract to your needs. Just remember to use the ticker of the assets you want to request the price for, and to **deploy your contract to the appropriate network, with the address of the oracle contract in that network**. Remember we have a [Quickstart](../for-devs/quickstart-solidity-hardhat) available that goes through the process of compiling and deploying a contract.
 
-### More examples
+### See also
 
-Refer to [Tellor’s documentation](https://docs.tellor.io/) for more examples of querying price feeds plus other data feeds available.
+- [Tellor’s documentation](https://docs.tellor.io/) demonstrates how to query price feeds and other data feeds.
+- [How to use Supra's price feed oracle](/for-devs/third-party-docs/Supra/use-supras-price-feed-oracle)
```

Comment:
This is at the same level as Tellor's section. I noticed the original contributor placed the links there, but it looks like they should go in a different section, right?
Maybe this "See also" section can be an h2 (##) ?

Reply:
Acknowledging receipt - will follow up!